packages/venia-ui/lib/components/Navigation/navigation.js

```diff
@@ -63,13 +63,13 @@ const Navigation = props => {
             <div className={bodyClassName}>
                 <CategoryTree
                     categoryId={categoryId}
-                    categories={categories}
                     onNavigate={handleClose}
                     setCategoryId={setCategoryId}
                     updateCategories={catalogActions.updateCategories}
                 />
             </div>
             <div className={classes.footer}>
+                <StoreSwitcher mobileView={true} />
```

Comment:
Usually we indicate mobile using media queries in css. I think it makes more sense to do it that way anyways. The component doesn't seem too complex, just a horizontal bar with an option/select.

Reply:
This allows not having two switchers in the same time in header and navigation panel.

Reply:
Is having two switchers rendered a concern?

Reply:
Resolved this on `header` and `navigation` components level with media queries for `.switchers` container